src/calculateRentalCost.js

```diff
@@ -28,7 +28,15 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  let totalCosts = days * 40;
```

Comment:
It's a good practice to use constants for magic numbers like 40, 50 and 20 to make the code more understandable. Consider using constants like `DAILY_RENTAL_RATE`, `DISCOUNT_7_DAYS` and `DISCOUNT_3_DAYS`. 😊